_posts/2022-05-06-grants-bitcoin-open-source.markdown

```diff
@@ -0,0 +1,41 @@
+
+_Disclaimer: Chaincode doesn't give grants. It's a challenging process and my gratitude goes out to all the grant programs that are doing their best to get this right. I wrote more about the giving side of things [here](/bitcoin/funding/funding-bitcoin-development/)._
+
+## Do the work and good things will happen
+
+Bitcoin funding is different than other open-source and other cryptocurrency projects. Getting a grant in bitcoin is pretty straightforward. You either need someone to vouch for you or you need to do work -- ideally both. While grant programs often have open applications, the secret to getting funding isn't much of a secret. Start doing the job for free. It establishes you as a contributor and proves your motivation to do the work. It shows that you are a good investment. Applicants will have much more success if they do the work and then apply. For most jobs, the applicant is trying to convince the employer that they are capable of doing the job. But the job-seeker has no idea what the work or environment is actually like. In open-source, one doesn't have to guess. Do the work. Demonstrate capability. Then ask for support.
+
+## How much to ask for
+
+The round number people tend to use is $100,000. This simplifies the process for grantors, so they don't have to make judgment calls on your standard of living or life circumstances. Some programs will adjust their grants based on need. $100,000 in San Francisco is very different than $100,000 in San Salvador. This requires more effort from both the grantor and the grantee to find the number that makes sense. It is also standard for most programs to allow you to take grants from other sources, but some require exclusivity. This is to prevent situations where people have stacked grants to the point they are making a higher than industry salary, which can appear greedy. I think you are usually fine if you are as transparent as possible with your needs and where you expect to source money.
+
+## Sustainability
+
+The goal of these grant programs is not to make you rich. If you are looking for financial upside, going the grant program route isn't for you. The list remains short of full-time roles for bitcoin open-source work, but they do exist. Grants are typically 12-month terms, but it is fair to ask about grant length and the renewal process. The majority of the grant turnover I see is not driven from the grantor side -- usually, either the grantee quits to do something else or they move to another sponsor because of better terms or to associate with a grant program with a better reputation.
```

Comment:
```suggestion
The goal of these grant programs is not to make you rich. If you are looking for financial upside, going the grant program route isn't for you. The list remains short of full-time roles for bitcoin open-source work, but they do exist. Grants are typically 12-month terms, but it is fair to ask about grant length and the renewal process. The majority of the grant turnover I see is not driven from the grantor side -- usually, either the grantee quits to do something else, or they move to another sponsor because of better terms or to associate with a grant program with a better reputation.
```